syfertext/doc.py

```diff
@@ -336,8 +338,11 @@ def get_token_vectors(self, excluded_tokens: Dict[str, Set[object]] = None) -> n
             if include_token:
```

Comment:
Since know the vectors are already tensors you don't need to explicitly convert them to torch tensor. Can you go through  doc.py and tokrn.py for such examples,(mostly go through functions which are related to vectors etc)

Reply:
@rajathpatel23 would you please consider @Nilanshrajput comment here

Reply:
@AlanAboudib sure, I am just confused with that function because the variable `token_vector` is list and was originally converted to numpy array due to which I decided to convert it to tensor because the return type of the function is changed to tensor from a numy array. However, let me it a try again today, review is more deeply. Also, I am using a Windows system due to which I am having problems configuring my environment to run tests locally which is the reason everything I commit is kind getting push to this PR.